src/network/os1/server/codec/game/UpdateSkillEncoder.ts

```diff
@@ -0,0 +1,16 @@
+import type Packet from '#/io/Packet.ts';
+
+import MessageEncoder from '#/network/server/codec/MessageEncoder.ts';
+
+import type UpdateSkill from '#/network/server/model/game/UpdateSkill.ts';
+
+export default class UpdateSkillEncoder extends MessageEncoder {
```

Comment:
Skills are called "Stats" (the trigger for a level up is advancestat for example) - this packet in particular is UPDATE_STAT